### PR TITLE
Updating information about Bower, Yarn, staging, alpha and beta

### PR DESCRIPTION
**Pull request by dotpointer**
Installation instructions for cosmoz-frontend has changed lately with the introduction of Yarn.

Alpha environment has also been introduced in favor of staging that has been phased out.

The documentation need to follow the changes, and this pull request is intended to start doing so.

I figure there are things missing in this pull request, so please review and fill in the blanks so I can add and update it.

### CODE REVIEW

File: docs/20-glossary.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ help others.
   `Gulp <http://gulpjs.com/>`_
```
Comment (JockeCK) [inline docs/20-glossary.rst]: Do we still need info about gulp, can't we just remove it completely instead?

**Comment by dotpointer** [comment]
Comment from plequang:

> I think that bower is still needed. Actually we should follow this https://www.polymer-project.org/2.0/docs/tools/polymer-cli#install, replacing npm with yarn

File: docs/20-glossary.rst (hunk 1 above)
Comment (dotpointer) [reply docs/20-glossary.rst]: I have now removed texts about gulp. Replaced gulp serve with yarn start where I found possible.

**Comment by dotpointer** [comment]
Installation instructions for bower are added back as indirectly requested by plequang.

**Review by Wurper — APPROVED** [review]

**Review by JockeCK — APPROVED** [review]
LGTM

**Pull request merged**